intelmq/bots/BOTS

```diff
@@ -653,6 +103,45 @@
                 "xmpp_server": "xmpp server",
                 "xmpp_user": "xmpp username"
             }
+        },
+        "AlienVault OTX": {
```

Comment:
> Do you still think that it should be ordered alphabetically rather than generic and specific?

The configuration has a lot of shortcomings, this PR addresses one of them. It does not solve the grouping/sorting ultimately, but it's better than before.

So, yes, I think we can drop the sorting now.

Please more opinions @certtools/intelmq-contributors concerning this change (not so much the technical questions, but the change in general).